ecs-agent/metrics/constants.go

```diff
@@ -46,6 +46,11 @@ const (
 	ACSDisconnectTimeoutMetricName = agentAvailabilityNamespace + ".ACSDisconnectTimeout"
 	TCSDisconnectTimeoutMetricName = agentAvailabilityNamespace + ".TCSDisconnectTimeout"
 
+	// ACS Session Metrics
+	acsStartSessionNamespace         = "ACSStartSession"
+	DiscoverPollEndpointDurationName = acsStartSessionNamespace + ".DiscoverPollEndpointDuration"
```

Comment:
Can we use a different namespace/prefix for this `DiscoverPollEndpointDurationName`? because discoverPollEndpoint is not for ACS only.

Reply:
oh right yes good point ill go ahead and change that